nimterop/ast2.nim

```diff
@@ -1611,16 +1640,17 @@ proc addDecl(gState: State, node: TSNode) =
 
   let
     start = getStartAtom(node)
+    commentNodes = gState.getCommentNodes(node)
```

Comment:
Does this work properly for multi-function declarations? Won't users have comments inline? Crazy example but who knows.

```c
// abc() is special
int abc(),
// def() is average
def(),
// ghi() is exotic
ghi();
```

Reply:
I'll have to test it out. Not sure actually!

Reply:
This was a problem. Fixed it!